state/mysql/mysql_test.go

```diff
@@ -308,61 +308,6 @@ func TestDeleteWithETag(t *testing.T) {
 	assert.Nil(t, err)
 }
 
-func TestReturnNDBResultsRowsAffectedReturnsError(t *testing.T) {
```

Comment:
Is there a way for us to preserve the unit tests? The mock tests can be useful for dev inner loop testing and with the exception of the TooManyRows check, the rest seem like they could be updated to check the updated behavior.

Reply:
I add more unit tests to serve the same purpose.